csunplugged/templates/resources/base-resource-pdf.html

```diff
@@ -0,0 +1,38 @@
+<html>
+<head>
+  <title>{{ filename }}</title>
+  <style>
+    @page {
+      margin: 10mm 10mm;
+      {% if paper_size == "a4" %}
+        size: A4;
+      {% elif paper_size == "letter" %}
+        size: letter;
+      {% endif %}
+    }
+    img.base-image {
+      {% if paper_size == "a4" %}
+        max-height: 257mm !important;
+      {% elif paper_size == "letter" %}
+        max-height: 239mm !important;
+      {% endif %}
+    }
+  </style>
+</head>
+<body>
+  <header class="fixed-running-element">
+    {{ header_text }}
+  </header>
+  {% for resource_image_data in resource_images %}
+    {% for resource_image in resource_image_data %}
+      <div class="base-image-container">
+        <img class="base-image" src="data:image/png;base64,{{ resource_image }}" />
+      </div>
+    {% endfor %}
+  {% endfor %}
+  <footer class="fixed-running-element">
+    {% load static %}
```

Comment:
Because it doesn't have to be, I put it next to the one place it is used, however for consistency I've placed it at the top.